lib/staleBuildChecks.js

```diff
@@ -18,16 +18,7 @@ const checkAndTagPRsWithOldBuilds = async (context) => {
     if (containsOldBuildLabel) {
       return;
     }
-    // Get last commit for pull request.
-    const {data: lastCommit} = await context.github.repos.getCommit(
-      context.repo({
-        ref: pullRequest.head.sha
-      })
-    );
-
-    const lastCommitDate = new Date(lastCommit.commit.author.date);
-    // Commit is older than 2 days
-    if (utils.MIN_BUILD_DATE > lastCommitDate) {
+    if (utils.checkBuild(pullRequest, context)) {
```

Comment:
There's no `utils.checkBuild` function.

Reply:
Extremely sorry sir for such mistake fixing it in latest commit.